mix.exs

```diff
@@ -30,7 +30,7 @@ defmodule SmartCityTest.MixProject do
   defp deps do
     [
       {:brod, "~> 3.8"},
-      {:smart_city, "~> 3.0"},
+      {:smart_city, "~> 3.0"},      
```

Comment:
Could you clean up this additional whitespace?

Reply:
Fixed!